package.json

```diff
@@ -61,9 +61,11 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-standard": "^5.0.0",
+    "fast-jwt": "^1.4.0",
     "fastify": "^3.1.1",
     "jest": "^27.0.3",
     "nock": "^13.0.2",
+    "path": "^0.12.7",
```

Comment:
Is it necessary?

Reply:
I use it in the tests. But it's not a must more a convenience, if tests are moved at least we don't need to update the path to the keys.

Reply:
path is a core module, you don't need to install it

Reply:
Removed it